app/javascript/components/public_health/actions/UpdateCaseStatus.js

```diff
@@ -102,6 +111,7 @@ class UpdateCaseStatus extends React.Component {
           isolation: this.state.isolation,
           monitoring: this.state.monitoring,
           monitoring_reason: this.state.monitoring_reason,
+          reasoning: this.state.isolation ? '' : [this.state.monitoring_reason, this.state.reasoning].filter(x => x).join(', '),
```

Comment:
Why is monitoring reason being included here?

Reply:
I copied this behavior from the CloseRecords file. I just asked the PO team and they want it kept this way.
"I would assume keep it, because they might not alway leave a note"

Reply:
just seems odd because you're submitting monitoring reason separately as well. resolving because the PO team okayed it